src/components/hr/hr.style.ts

```diff
@@ -1,8 +1,10 @@
 import styled from "styled-components";
-import { margin } from "styled-system";
+import { margin, MarginProps } from "styled-system";
 import baseTheme from "../../style/themes/base";
 
-const StyledHr = styled.hr`
+type StyledHrProps = React.HTMLAttributes<HTMLElement> & MarginProps;
```

Comment:
I'm not sure if `React.HTMLAttributes<HTMLElement>` is necessary here
https://styled-components.com/docs/api#using-custom-props

Reply:
Yeah you're right @mkrds - I hadn't noticed that in the docs. I've changed it and tested it as well 👍🏻